infra/blueprint-test/examples/tf_vet_valid_config/main.tf

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+resource "google_project_service" "project" {
+  project                    = var.project_id
+  service                    = "cloudresourcemanager.googleapis.com"
```

Comment:
Lets collapse both TF configs for valid/violation into a single config with a service TF input which we set via the test framework.

Reply:
configs joined together